beater/beater.go

```diff
@@ -16,6 +17,7 @@ type beater struct {
 
 // Creates beater
 func New(_ *beat.Beat, ucfg *common.Config) (beat.Beater, error) {
+	cfgwarn.Beta("You are using an beta release of apm-server.")
```

Comment:
`a beta release`, again does it make sense to use `APM Server instead?

Reply:
will change it accordingly.